packages/material-ui/src/ButtonBase/ButtonBase.js

```diff
@@ -229,7 +236,7 @@ class ButtonBase extends React.Component {
   render() {
     const {
       action,
-      buttonRef,
+      buttonRef: buttonRefProp,
```

Comment:
Do we need to rename the prop?

Reply:
Right, we don't. Going to extract the ref callback into a class instance function though so we don't run it on every render.